### PR TITLE
frontend for profile page

### LINKED ISSUE #1: Profile Page Frontend
Creating the profile page for users and its related features (i.e. personalization, image, description, redirects to other pages)

### PR DESCRIPTION
**Pull request by leed85**
This is the current frontend for the profile page, no backend/functionality, the functionality will be in the future pull request

### CODE REVIEW

**Comment by thecatfour** [comment]
Your frontend directory for src/ is slightly off. This is what our convention currently looks like:
```
src/
    app/
         profile/
             page.js
    components/
         profile-components/
             sidebar.js
             sidebar.css
```
Basically, you components directory is in app/ when it's supposed to be in src/. Also, you are using <filename>.css instead of <filename>.module.css, which is what we have been doing for most components. However, I personally think its fine to leave the css files as they are currently.

File: frontend/src/app/profile/page.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+              type="file"
+              id="profilePicInput"
+              accept="image/*"
+            //   onChange={handleImageChange}
+              className="profilePicInput"
+              style={{ display: 'none' }}
+            />
+            <div className="userDetails">
+              <div className='sectioncount'>
+                <div className='counter'>
```
Comment (thecatfour) [inline frontend/src/app/profile/page.js:50]: This is a nitpick, but is there a reason you use single quotes ' for some classNames while using double quotes " for others?

**Comment by leed85** [comment]
> Your frontend directory for src/ is slightly off. This is what our convention currently looks like:
> 
> ```
> src/
>     app/
>          profile/
>              page.js
>     components/
>          profile-components/
>              sidebar.js
>              sidebar.css
> ```
> 
> Basically, you components directory is in app/ when it's supposed to be in src/. Also, you are using .css instead of .module.css, which is what we have been doing for most components. However, I personally think its fine to leave the css files as they are currently.

Thanks Ill update so that it matches the current structure

File: frontend/src/app/profile/page.js (hunk 1 above)
Comment (leed85) [reply frontend/src/app/profile/page.js:50]: This is just an error on my part, I was planning on just single quotes but I guess I accidentally switched quotes in some places, thanks

File: frontend/src/app/components/sidebar.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <div className='sidebar'>
+            <div className='logo'>
+                <Image
+                    width={250} 
+                    height={100} 
+                    src="/UNLV.png"
+                    className='logo'
+                />
+            </div>
+            <Link href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" className='link'>Home</Link>
```
Comment (thecatfour) [inline frontend/src/app/components/sidebar.js:17]: I can't seem to click on this link. Can someone else check it and tell me where it leads?

**Comment by leed85** [comment]
I'm am going to temporarily close this pull request due to merge conflicts, will create 2 separate pull requests, separate one with the sidebar and other with the profile page.

**Pull request closed without merging**